runner.go

```diff
@@ -235,6 +240,14 @@ func handleCommand(hndlr *cmdHandler) (int, []byte, float64, error) {
 		emitEvent(title, body, hndlr.opts.Label, alertType, hndlr)
 	}
 
+	// Passthru stdin/stdout to controlling tty
+	if hndlr.opts.Passthru {
+		fmt.Printf("%v", string(out))
+		fmt.Fprintf(os.Stderr, "%v", string(c.Bytes()))
+		// no need to reprint stdout/stderr on bailOut
+		hndlr.opts.Sensitive = true
+	}
```

Comment:
That's correct. The use case I had in mind was cron jobs with MAILTO enabled. It's often the case where only errors should be emailed, separating the streams on `passthru` would allow one to run something like:

```
cronner -l job_name -p -- my_script.sh >/dev/null
```
This would still pass stderr to the parent process and initiate emails only on error.